onnx/helper.py

```diff
@@ -482,6 +486,59 @@ def make_tensor_value_info(
     return value_info_proto
 
 
+def make_sparse_tensor_value_info(
+        name,  # type: Text
+        elem_type,  # type: int
+        shape,  # type: Optional[Sequence[Union[Text, int]]]
+        doc_string="",  # type: Text
+        shape_denotation=None,  # type: Optional[List[Text]]
+):  # type: (...) -> ValueInfoProto
+    """Makes a ValueInfoProto based on the data type and shape."""
+    value_info_proto = ValueInfoProto()
+    value_info_proto.name = name
+    if doc_string:
+        value_info_proto.doc_string = doc_string
+
+    sparse_tensor_type_proto = value_info_proto.type.sparse_tensor_type
+    sparse_tensor_type_proto.elem_type = elem_type
+
+    sparse_tensor_shape_proto = sparse_tensor_type_proto.shape
```

Comment:
Should this be inside the if-then branch? If shape is None, I guess it means an unknown shape, right? If we create a shape here, wouldn't that become a scalar of rank zero?